halo/genutil/evm/predeploys/predeploys.go

```diff
@@ -156,6 +162,17 @@ func setSlashing(db *state.MemDB) error {
 	return setPredeploy(db, slashing, slashingCode, bindings.SlashingStorageLayout, storage)
 }
 
+// setGovernance sets the Governance predeploy.
+// TODO: do we really need this?
+func setGovernance(db *state.MemDB, owner common.Address) error {
+	storage := state.StorageValues{
+		"_initialized": uint8(1), // disable initializer
```

Comment:
* Do we need the storage values for governance, probably are different if at we have them at all